src/Libs/Gd.php

```diff
@@ -258,4 +269,46 @@ public function rotate($angle)
 
         $this->image = $image;
     }
+
+    /**
+     * @param string $watermark Watermark image object
```

Comment:
`@param string $watermark Watermark image object` is not correct, is an object or a string?